### ISSUE TITLE
 Too many problem and never run forwever

### ISSUE THREAD

**Issue by two-tigger** [dmar-bonn/ipp-marl#3]

**Comment by Sun-Xinnnnn** [comment]
No, he can run the result normally, and thank the author very much for his contribution to the field, I do not hope to rise to the problem of this code, because the code written by the author is correct, only some scenarios need to be changed, but the default parameters can run

**Comment by tuanmanh1410** [comment]
> No, he can run the result normally, and thank the author very much for his contribution to the field, I do not hope to rise to the problem of this code, because the code written by the author is correct, only some scenarios need to be changed, but the default parameters can run

Hi , can you guide me how to run this source code (main.py file) ? Thank yo in advance
I tried but it shows the error from the init step for miss installing **marl_framework**. 
It seem that I should make setup to install custom library,
